### PR TITLE
 believe!: empty!

### PR DESCRIPTION
**Pull request by FuneralforaFriend**
I am inevitable!

### CODE REVIEW

**Comment by godoty** [comment]
hi

File: test.py [hunk 1]
```diff
@@ -0,0 +1 @@
+print("I am inevitable")
```
Comment (godoty) [inline test.py:1]: I couldn't understand

**Review by godoty — COMMENTED** [review]
Nah

File: test.py (hunk 1 above)
Comment (JS0303) [reply test.py:1]: OK

**Pull request merged**